framework/db/oci/QueryBuilder.php

```diff
@@ -162,6 +162,27 @@ public function addForeignKey($name, $table, $columns, $refTable, $refColumns, $
     }
 
     /**
+     * For each return column a param will be added to $returnParams.
+     * They should have variables bound before executing the query.
+     * @inheritdoc
+     */
+    public function insertReturning($table, $columns, &$params, $returnColumns = null, &$returnParams = null)
+    {
+        $schema = $this->db->getSchema();
```

Comment:
Oh, you meant the other way, trying to do `queryOne` for Oracle. I didn't try that yet.